products/cbbr/03_spatial.sh

```diff
@@ -17,7 +17,7 @@ echo "Assign geometries from facilities data ..."
 run_sql_file sql/spatial_facilities.sql
 
 echo "Assign geometries from manual shapefiles ..."
-run_sql sql/spatial_manual.sql
```

Comment:
(just curious) did `run_sql` fail quietly or (hopefully) throw an error and stop the build?

Reply:
It errored